src/js/angular/core/filters/bytes-filter.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * @ngdoc filter
+ * @name bytes
+ * @module graphdb.framework.core.filters.bytes
+ * @description Converts a number of bytes into a human-readable string with units
+ * @param {number} bytes The number of bytes to convert
+ * @param {number} [precision=1] Number of decimal places to display
+ * @returns {string} Formatted string with appropriate unit (bytes, kB, MB, GB, TB, PB)
+ * @example
+ * // returns "1.5 MB"
+ * {{ 1572864 | bytes }}
+ *
+ * // returns "1.50 MB"
+ * {{ 1572864 | bytes:2 }}
+ */
+angular.module('graphdb.framework.core.filters.bytes', [])
+    .filter('bytes', function () {
```

Comment:
There is similar logic in `heap-memory-chart.js`

Can something be reused?

Reply:
Well, It's possible to reuse the filter I added in the chart and not the other way round because the convertor from the chart is kind of tied with it. Anyway. I don't wan't to deal with refactoring in other modules in this MR.